test/elastic_helpers.rb

```diff
@@ -0,0 +1,92 @@
+# rubocop:disable Metrics/MethodLength
+require 'fakeweb'
```

Comment:
This module needs review - it works for testing but not sure if its up to convention. It was the most 'clever' way I could come up for emulating ES in cucumber tests for now